cmd/podman/tree.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containers/libpod/cmd/podman/libpodruntime"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+var (
+	treeFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "whatrequires",
```

Comment:
`whatrequires` is not a great fit — I realize it is supposed to be reminiscent of `rpm -q --whatrequires`, but unlike RPM, it is perfectly possible to remove (well, untag) the parent image; the child image does not “require” the parent in any meaningful sense.